job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContextSpec.groovy

```diff
@@ -3042,21 +3043,27 @@ class PublisherContextSpec extends Specification {
             tag 'tag'
             shouldWaitForRundeckJob()
             shouldFailTheBuild false
+            shouldIncludeRundeckLogs()
         }
 
         then:
         Node rundeckNode = context.publisherNodes[0]
         rundeckNode.name() == 'org.jenkinsci.plugins.rundeck.RundeckNotifier'
-        rundeckNode.jobId[0].value() == 'jobId'
```

Comment:
During the new feature development I came across a situation (after I renamed a node name) when assertion like that failed. The error message was:

```
java.lang.NullPointerException: Cannot invoke method value() on null object
```

From the error message I know that there wasn't node with given name, but I cannot say why - developer has to debug it or add some additional logging statement before. 

After my change the error message became much more meaningful:
```
rundeckNode.includeRundeckLogsWrongName[0]?.value() == true
|           |                  |    |       |
|           []                 null null    false
org.jenkinsci.plugins.rundeck.RundeckNotifier[attributes={}; value=[jobId[attributes={}; value=jobId],
options[attributes={}; value=], nodeFilters[attributes={}; value=], tag[attributes={}; value=], 
shouldWaitForRundeckJob[attributes={}; value=true], shouldFailTheBuild[attributes={}; value=false],
includeRundeckLogs[attributes={}; value=true]]]
```

It is in some sense a boiler plate code, but unfortunately currently it [doesn't support](https://github.com/spockframework/spock/pull/49) it. I personally prefer a need to add additional `?` to get verbose error message in case of regression.